clientserver/src/mill/clientserver/Server.scala

```diff
@@ -106,7 +120,13 @@ class Server[T](lockBase: String,
 
     t.interrupt()
     t.stop()
-    clientSocket.close()
+
+    if (ClientServer.isWindows) {
+      // Closing Win32NamedPipeSocket can take a second or more!
```

Comment:
If the previous `Win32NamedPipeSocket` hasn't fully closed, is it really OK for us to return from `handleRun` and possibly open a new `Win32NamedPipeSocket`? Maybe it is, but if so you should add a comment because it's not obvious from the code that having two sockets open at the same time is OK

Reply:
Yes, this is a hack. I tested this by calling mill (client) several times on quick successions and it seems to work OK (in my machine); I wonder whether closing the server socket helps speed up the process. Without that, there is a very perceivable lag (often ~3-5 seconds) between the last output from mill server and when the mill client exits. I can put in more comments.